### ISSUE TITLE
Replace OpenHW Group logos on OpenHW GRoup and Eclipse website

### ISSUE THREAD

**Issue by michelleclancy** [openhwgroup/openhwgroup.org#491]
The OpenHW Group logos have now been updated with a registered trademark.  The updated logos can be found here: 
https://drive.google.com/drive/folders/1SkW5v_nRQTvXj6lzX4SPt5U4P670_onN?usp=sharing 

We need to replace all the logos on the OpenHW website and on the Eclipse site - 

OpenHW Website
        All Pages:
Website Header Logo – Replace with “OpenHW Landscape” logo file
Website Footer Logo – Replace with “OpenHW Landscape” logo file
Footer Text– Add in a ® symbol, along with the following text: “Copyright © OpenHW Group®. All rights reserved. OpenHW, OpenHW Group, CORE-V, and the OpenHW Group and CORE-V logos are trademarks of OpenHW Group.
 
        [Homepage:](https://www.openhwgroup.org/)
CORE-V Family Section (middle of page) – Replace with “CORE-V Portrait” logo file
 
        Membership – Become a Member
[Member Contact Form](https://www.openhwgroup.org/membership/openhw-membership-contact-form.pdf)
 
                [Downloads](http://downloads.openhwgroup.org/)
Website Header Logo
 

Eclipse Foundation Website
[Eclipse Logos and Artwork](https://www.eclipse.org/org/artwork/#openhw)
Working Groups Logos – All OpenHW Group logos that are linked to this file

**Comment by ZacharySabourin** [comment]
> The OpenHW Group logos have now been updated with a registered trademark. The updated logos can be found here: https://drive.google.com/drive/folders/1SkW5v_nRQTvXj6lzX4SPt5U4P670_onN?usp=sharing
> 
> We need to replace all the logos on the OpenHW website and on the Eclipse site -
> 
> OpenHW Website All Pages: Website Header Logo – Replace with “OpenHW Landscape” logo file Website Footer Logo – Replace with “OpenHW Landscape” logo file Footer Text– Add in a ® symbol, along with the following text: “Copyright © OpenHW Group®. All rights reserved. OpenHW, OpenHW Group, CORE-V, and the OpenHW Group and CORE-V logos are trademarks of OpenHW Group.

These changes should be available for preview here: https://deploy-preview-494--eclipsefdn-openhwgroup.netlify.app/

> ```
>     [Homepage:](https://www.openhwgroup.org/)
> ```
> 
> CORE-V Family Section (middle of page) – Replace with “CORE-V Portrait” logo file

Did you mean the "OPENHW-portrait"? There is no CORE-V portrait provided in the shared GDRIVE.

> ```
>     Membership – Become a Member
> ```
> 
> [Member Contact Form](https://www.openhwgroup.org/membership/openhw-membership-contact-form.pdf)
> 
> ```
>             [Downloads](http://downloads.openhwgroup.org/)
> ```

I'm not quite sure what the requirements are here. Do you want the logo on the pdf to be updated? If so, you will have to update the file on your end and provide me with the new PDF.

> Website Header Logo
> 
> Eclipse Foundation Website [Eclipse Logos and Artwork](https://www.eclipse.org/org/artwork/#openhw) Working Groups Logos – All OpenHW Group logos that are linked to this file

I have created an issue in GitLab https://gitlab.eclipse.org/eclipsefdn/it/websites/eclipse.org/-/issues/179 to track the file changes. 
Would you like me to ZIP the contents of the shared GDRIVE and replace the old ZIP file on Eclipse Foundation Website? The old ZIP file contains Europe and Asia versions as well. The shared GDRIVE only contains non-Asia/Europe versions.

**Comment by autumnfound** [comment]
@michelleclancy please provide feedback as we have a few questions and would love to get this live!

**Comment by michelleclancy** [comment]
@autumnfound  the website logos at the top are perfect.  The CORE- V logo at the bottom of the page needs to be updated. here are the logos - https://drive.google.com/drive/folders/1Tt2dpVVqMsnyE1XITl1L8Xc7WG1uu
<img width="1284" alt="Screen Shot 2022-08-22 at 12 35 33 PM" src="https://user-images.githubusercontent.com/88848029/185974101-21d913be-4fd3-44cb-a254-abd06fa61f95.png">
6qm
here is the logo that needs updating  
as for the Membership form, we will update the PDF and send over a new file

**Comment by autumnfound** [comment]
I merged in the current changes as having at least the main part of this request covered is more important than having it all done in one swoop I think. @michelleclancy please let us know if you need support for these assets.

**Comment by autumnfound** [comment]
Do you need support for the PDF/zip files? These assets still have no registered mark and thus weaken the compliance/usage of our trademark, and we should get these updated ASAP

**Comment by autumnfound** [comment]
> Do you need support for the PDF/zip files? These assets still have no registered mark and thus weaken the compliance/usage of our trademark, and we should get these updated ASAP

@chrisguindon has let me know that we've reached out to our designer to help provide these assets. We will ping here again when we have a preview of this content!

**Comment by chrisguindon** [comment]
For tracking purposes, the zip file update is being done via https://gitlab.eclipse.org/eclipsefdn/it/websites/eclipse.org/-/issues/179

**Comment by ZacharySabourin** [comment]
@michelleclancy The artwork update is now live: [eclipse.org/org/artwork#openhw](https://www.eclipse.org/org/artwork/#openhw)

**Comment by michelleclancy** [comment]
@ZacharySabourin thank you!! looks good

**Comment by ZacharySabourin** [comment]
@chrisguindon I believe we are still missing the [Member Contact Form](https://www.openhwgroup.org/membership/openhw-membership-contact-form.pdf)

**Comment by chrisguindon** [comment]
> @chrisguindon I believe we are still missing the [Member Contact Form](https://www.openhwgroup.org/membership/openhw-membership-contact-form.pdf)

@ZahraFazli do you own the master document for these membership contact form? https://www.openhwgroup.org/membership/openhw-membership-contact-form.pdf

If so, can you provide a new version with the logo that include the registered trademark?

**Comment by ZahraFazli** [comment]
Hi @chrisguindon what do you mean by the Master version of the Contact Us form?
I have Member Contact Forms of the members but they do not have logos attached. Logos come separately.

**Comment by chrisguindon** [comment]
@ZahraFazli I meant the original document. 

tldr: The OpenHW Group logos have now been updated with a registered trademark. 

https://www.openhwgroup.org/membership/openhw-membership-contact-form.pdf is a pdf. We need someone to update the original doc to replace the logo so that we can update the website.

**Comment by chrisguindon** [comment]
@ZahraFazli Any updates on this?

**Comment by ZahraFazli** [comment]
I have updated the Master file. Attached is the PDF version  that also needs to be uploaded on the website here: 
[OpenHW Group - Membership Information Form.docx.pdf](https://github.com/openhwgroup/openhwgroup.org/files/9778087/OpenHW.Group.-.Membership.Information.Form.docx.pdf)
(https://www.openhwgroup.org/membership/openhw-membership-contact-form.pdf)

**Comment by ZacharySabourin** [comment]
@ZahraFazli @michelleclancy Here is a preview of the updated PDF: https://deploy-preview-531--eclipsefdn-openhwgroup.netlify.app/membership/openhw-membership-contact-form.pdf

**Comment by ZahraFazli** [comment]
Looks good to me.  Thanks!

**Comment by DBees** [comment]
This is incorrect. The file had been changed to include MWG and TWG representatives. The new version removes that. Did you remove that on purpose @ZahraFazli ?

**Comment by DBees** [comment]
@ZahraFazli I have resent you the original of this form from Feb 2022 - please update that one with the registered trademark. Thanks. Duncan

**Comment by ZahraFazli** [comment]
Here's the updated form. 
[Copy of OpenHW Group - Membership Information Form.docx.pdf](https://github.com/openhwgroup/openhwgroup.org/files/9889757/Copy.of.OpenHW.Group.-.Membership.Information.Form.docx.pdf)

**Comment by ZahraFazli** [comment]
@DBees please see the updated form attached above

**Comment by DBees** [comment]
@ZahraFazli can you confirm the form editing is complete so we can post this and close the issue?

**Comment by ZahraFazli** [comment]
Here's the final version, approved by Rick and Duncan. Thanks
[OpenHW Group - Membership Information Form.pdf](https://github.com/openhwgroup/openhwgroup.org/files/10032915/OpenHW.Group.-.Membership.Information.Form.pdf)

**Comment by ZacharySabourin** [comment]
@DBees Here is a preview with the updated PDF: https://deploy-preview-531--eclipsefdn-openhwgroup.netlify.app/membership/openhw-membership-contact-form.pdf

**Comment by DBees** [comment]
looks good, thank you
